README.md

```diff
@@ -12,6 +12,8 @@ This handbook should be the starting point for any new team members. It provides
 
 * **Culture:** We value a positive and diverse company culture, and will always strive to adapt our culture in order to make this a more welcoming place to be.
 
+* **Continuous Improvement:** We value continuous improvement, supporting it with a blame free culture, and seek to empower ourselves to be able to reflect and improve.
```

Comment:
The "blame free culture" sets off an alarm for me, not sure why. The best way I can articulate how it makes me feel is: imagine going to a swimming pool and there was a sign saying "there's DEFINITELY no man eating sharks in here".  I don't think I'd go in!

Plus, I think that point is implicitly covered by the People value - having real respect for the people you work with should curb the rise of a potential blame culture.

Maybe "We value continuous improvement, and seek to empower ourselves and the people around us with the opportunity to reflect and improve."?

Reply:
Yeah I see what you mean. Best to reaffirm the positive in the value statement rather than dwell on the negative. Thanks Scott!